js/packages/common/src/contexts/accounts.tsx

```diff
@@ -146,10 +146,6 @@ export const cache = {
     parser?: AccountParser,
     isActive?: boolean | undefined | ((parsed: any) => boolean),
   ) => {
-    if (obj.data.length === 0) {
```

Comment:
Removing this allows us to fetch and watch the SOL balance of a wallet, which may be empty.